src/Administration/Controller/AdminTagController.php

```diff
@@ -24,7 +25,12 @@ public function __construct(private readonly FilterTagIdsService $filterTagIdsSe
     {
     }
 
-    #[Route(path: '/api/_admin/tag-filter-ids', name: 'api.admin.tag-filter-ids', defaults: ['_acl' => ['tag:read'], '_entity' => 'tag'], methods: ['POST'])]
+    #[Route(
+        path: '/api/_admin/tag-filter-ids',
+        name: 'api.admin.tag-filter-ids',
+        defaults: [PlatformRequest::ATTRIBUTE_ACL => ['tag:read'], PlatformRequest::ATTRIBUTE_ENTITY => TagDefinition::ENTITY_NAME],
+        methods: ['POST']
```

Comment:
```suggestion
        methods: [Request::METHOD_POST]
```

if you are already on it, we should make it really consistent :grin: